lib/node_modules/@stdlib/blas/base/ssyr/README.md

```diff
@@ -183,21 +183,33 @@ console.log( A );
 ### Usage
 
 ```c
-TODO
+#include "stdlib/blas/base/ssyr.h"
 ```
 
-#### TODO
+#### c_ssyr( order, uplo, N, alpha, \*X, strideX, \*A, LDA )
 
-TODO.
+Performs the symmetric rank 1 operation `A = α*x*x^T + A`
 
 ```c
-TODO
+float A[] = { 1.0f, 0.0f, 0.0f, 2.0f, 1.0f, 0.0f, 3.0f, 2.0f, 1.0f };
+float x[] = { 1.0f, 2.0f, 3.0f };
+
+c_ssyr( 102, 121, 3, 1.0f, x, 1, A, 3 );
 ```
 
-TODO
+The function accepts the following arguments:
+
+-   **order**: `[in] CBLAS_LAYOUT` storage layout.
+-   **uplo**: `[in] CBLAS_UPLO` specifies whether the upper or lower triangular part of the symmetric matrix `A` should be referenced.
+-   **N**: `[in] CBLAS_INT` number of elements along each dimension of `A`.
+-   **alpha**: `[in] float` scalar.
+-   **X**: `[in] float*` input array.
+-   **strideX**: `[in] CBLAS_INT` index increment for `X`.
+-   **A**: `[inout] float*` input matrix.
+-   **LDA**: `[in] CBLAS_INT` stride of the first dimension of `A` (a.k.a., leading dimension of the matrix `A`).
 
 ```c
-TODO
+void c_ssyr( const CBLAS_LAYOUT order, const CBLAS_UPLO uplo, const CBLAS_INT N, const float alpha, float *X, const CBLAS_INT strideX, float *A, const CBLAS_INT LDA )
 ```
 
```

Comment:
@aman-095 Looks like we still need to add the ndarray API docs, correct?

Reply:
Yes, I will push it up soon.